.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "frontend"]
 	path = frontend
-	url = https://github.com/tfd-ed/sarpheab-client
+	url = https://github.com/MengTy199/sarpheab-client
```

Comment:
It should be officially repo for client https://github.com/tfd-ed/sarpheab-client